distributed/client.py

```diff
@@ -770,6 +773,8 @@ def __init__(
         extensions=DEFAULT_EXTENSIONS,
         direct_to_workers=None,
         connection_limit=512,
+        preload=None,
+        preload_argv=None,
         **kwargs,
```

Comment:
When creating a `LocalCluster` implicitly, these `kwargs` are forwarded to the `LocalCluster` constructor. Can you check is `preload` and `preload_argv` are valid inputs for `LocalCluster`? I'm wondering if there is existing user code that already is passing `preload` / `preload_argv` to `Client`

Reply:
Great point. Existing `preload`/`preload_argv` would be passed as `worker_kwargs` into the worker instance, I think. That could potentially be a problem, this change would unexpectedly redirect the `preload` to the `Client` instead. This is really tricky, I'm not sure how to deal with it. I can think of a few options right away:

1. Name these `client_preload`/`client_preload_argv`
   a. Possibly add a deprecation cycle to require `preload`s that are intended for `worker`s or the `scheduler` to be called `worker_preload`/`scheduler_preload`. (There is an existing `scheduler_kwargs` keyword in `LocalCluster`)
2. Hope that no one is passing a `preload` to the worker on a `LocalCluster` (no changes needed)
3. Change `LocalCluster` to not capture all the extra kwargs that it gets, and have `worker_kwargs` be a single argument that can be passed (like `scheduler_kwargs`)

Do you have any suggestions here?